src/app/components/Footer/index.jsx

```diff
@@ -1,27 +1,119 @@
 import React from 'react';
 import styled from 'styled-components';
 import {
-  C_STORM,
+  C_GREY,
   C_WHITE,
   FF_NEWS_SANS_REG,
+  GEL_SPACING,
   GEL_SPACING_DBL,
 } from '../../lib/constants/styles';
 
 const StyledFooter = styled.footer`
-  background-color: ${C_STORM};
+  background-color: ${C_GREY};
   font-family: ${FF_NEWS_SANS_REG};
-  padding: ${GEL_SPACING_DBL}px;
+  padding: ${GEL_SPACING_DBL}px ${GEL_SPACING}px;
+`;
+
+const StyledList = styled.ul`
+  border-bottom: 1px solid ${C_WHITE};
+  display: grid;
+  grid-auto-flow: column;
+  grid-template-columns: repeat(2, 50%);
+  grid-template-rows: repeat(4, 50%);
+  list-style-type: none;
+  margin: 0 0 ${GEL_SPACING}px 0;
+  padding: 0 0 ${GEL_SPACING}px 0;
+  > li:first-child {
+    border-bottom: 1px solid ${C_WHITE};
+    grid-column: 1/3;
+    margin-bottom: ${GEL_SPACING}px;
+    @supports not (display: grid) {
+      width: 100%;
+    }
+  }
+`;
+
+const StyledListItem = styled.li`
+  min-width: 50%;
+  padding: ${GEL_SPACING}px;
+  @supports not (display: grid) {
+    padding: ${GEL_SPACING}px 0;
+    display: inline-block;
+  }
+`;
+
+const StyledLink = styled.a`
+  color: ${C_WHITE};
+  text-decoration: none;
+
+  &:hover,
+  &focus {
+    text-decoration: underline;
+  }
 `;
 
 const StyledParagraph = styled.p`
   color: ${C_WHITE};
+  padding: ${GEL_SPACING}px 0 0 0;
+  margin: 0;
 `;
 
+const listItems = [
```

Comment:
This data is here in the component for now. I can move it out into a Footer Container in another PR.

Reply:
This needs an issue